### DISCUSSION TITLE
Lesson 11 create_metadata.py doesn't work on rinkeby but works on kovan

### DISCUSSION THREAD

**Discussion by ChristianG77** [smartcontractkit/full-blockchain-solidity-course-py#234] [Q&A] [answered]
Hi,

Any guidance on what to check for when running scripts on create_metadata.py and doesn't work on rinkeby but works on kovan?

PS C:\Users\chris\Dropbox\00_CODE\NFT-DEMO> brownie run scripts/advanced_collectible/create_metadata.py --network rinkeby
INFO: Could not find files for the given pattern(s).
Brownie v1.16.4 - Python development framework for Ethereum

NftDemoProject is the active project.

Running 'scripts\advanced_collectible\create_metadata.py::main'...
  File "C:\Users\chris\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\_cli\run.py", line 49, in main
    return_value, frame = run(
  File "C:\Users\chris\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\project\scripts.py", line 103, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File ".\scripts\advanced_collectible\create_metadata.py", line 10, in main
    advanced_collectible = AdvancedCollectible[-1]
  File "C:\Users\chris\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\network\contract.py", line 160, in __getitem__
    return self._contracts[i]
IndexError: list index out of range

PS C:\Users\chris\Dropbox\00_CODE\NFT-DEMO> brownie run scripts/advanced_collectible/create_metadata.py --network kovan
INFO: Could not find files for the given pattern(s).
Brownie v1.16.4 - Python development framework for Ethereum

NftDemoProject is the active project.

Running 'scripts\advanced_collectible\create_metadata.py::main'...
You have created 1 collectibles!
Creating Metadata file: ./metadata/kovan/0-PUG.json
https://ipfs.io/ipfs/QmSsYRx3LpDAb1GZQm7zZ1AuHZjfbPkD6J7s9r41xu1mf8?filename=pug.png
https://ipfs.io/ipfs/QmZKUjW9j5tBG336Y4NLxnGuw3J96VqU76C2DHF9KPS4c2?filename=0-PUG.json

**Comment by PatrickAlphaC** [comment, marked as answer]
You need to deploy the contract to rinkeby first!

```
brownie run scripts/advanced_collectible/deploy_and_create.py --network rinkeby
```

**Reply by ChristianG77** [reply]
Thanks @PatrickAlphaC!
